Sources/LiveKit/Views/InternalSampleBufferVideoRenderer.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 LiveKit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import WebRTC
+
+internal class InternalSampleBufferVideoRenderer: NativeView {
```

Comment:
What does `Internal` reference? or should we call it `AVSampleBufferDisplayLayerRenderer` ?

Reply:
This is not exposed, the `VideoView` uses it internally as a subview.

Reply:
got it, sounds good!